src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/MissingOverride/context/Test.java

```diff
@@ -0,0 +1,24 @@
+package TreeWalker.MissingOverride;
+
+public class Test {
+
+    /** {@inheritDoc} */
+    public void test1() {  // violation context
+
+    }
+
+    /** {@inheritDoc} */
```

Comment:
Second Problem: When I run ` java $RUN_LOCALE -jar checkstyle-8.34-SNAPSHOT-all.jar -T Test.java`, I get :

```
|--METHOD_DEF -> METHOD_DEF [11:4]
    |   |--MODIFIERS -> MODIFIERS [11:4]
    |   |   |--BLOCK_COMMENT_BEGIN -> /* [10:4]
    |   |   |   |--COMMENT_CONTENT -> * {@inheritDoc}  [10:6]
    |   |   |   `--BLOCK_COMMENT_END -> */ [10:21]
    |   |   `--LITERAL_PUBLIC -> public [11:4]
    |   |--TYPE -> TYPE [11:11]
    |   |   `--LITERAL_VOID -> void [11:11]
    |   |--IDENT -> test2 [11:16]
    |   |--LPAREN -> ( [11:21]
    |   |--PARAMETERS -> PARAMETERS [11:22]
    |   |--RPAREN -> ) [11:22]
    |   `--SLIST -> { [11:24]
    |       |--SINGLE_LINE_COMMENT -> // [11:27]
    |       |   `--COMMENT_CONTENT ->  violation context\n [11:29]
    |       `--RCURLY -> } [13:4]
```
So, MODIFIERS is child node of METHOD_DEF, so I thought that this context strategy can be solved by previous solution, but in fact that in TreeWalkerAuditEvent event, the AST does not include comments, modifiers, so how do we solve this situation?

Reply:
if check is Comments aware, it executes in separate Treewalking phase when comments are appended to Tree.
[MissingOverrideCheck.java](https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java) is not such Check. So it will not see see such tokens at all, and his validation does not need it for now.

check that override
https://github.com/checkstyle/checkstyle/blob/a72fc7227a79d010a52b89f022b4b4146ecb0de5/src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractCheck.java#L85
will have comments tokens during executions and during filtering. 

Please consider to think if inthis case strategy from https://github.com/checkstyle/patch-filters/pull/130#discussion_r460447179 might help.

Reply:
I don't understand your question. AST node from TreeWalkerAuditEvent is not METHOD_DEF for this check, right?

Reply:
@baratali AST node from TreeWalkerAuditEvent is  METHOD_DEF for this check, the problem is that the AST does not include comments, modifiers, so we can not get this violation through `getChildAstLineNo`

Reply:
@romani this https://github.com/checkstyle/patch-filters/pull/130#discussion_r460447179 might not help, because AST node from TreeWalkerAuditEvent is METHOD_DEF for this check, but the AST does not include comments, modifiers